src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentClient.kt

```diff
@@ -226,4 +251,16 @@ class CodyAgentClient(private val project: Project, private val webview: NativeW
 
     return saveFileFuture
   }
+
+  @JsonNotification("window/didChangeContext")
+  fun window_didChangeContext(params: Window_DidChangeContextParams) {
+    println(params)
+  }
+
+  @JsonNotification("globalStorage/didChange")
```

Comment:
Let's not sniff global storage for this. We should be listening to the configuration observable and pushing a semantic, application-level message to the IDE.